addons/hr_recruitment/static/src/scss/hr_job.scss

```diff
@@ -58,10 +54,6 @@
             .o_hr_recruitment_kanban_card_large_screen {
                 display: none;    
             }
-
-            .o_kanban_record {
-                margin: 0 0 ($border-width * -1);
-            }
         }
 
         &.o_kanban_ungrouped {
```

Comment:
as we already have one in the block above, please double check we still have all the styling we want (we could also merge it there)